src/styles/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: $border-width dotted $border-color;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the variables for colors and sizes should be used in all SCSS files. Here, it seems that you've missed defining a variable for the dotted border style. Consider adding a variable for it in your variables.scss file for better maintainability and consistency with the rest of your styles.